.NET/Microsoft.Recognizers.Text.DateTime/Utilities/MergedParserUtil.cs

```diff
@@ -520,6 +520,18 @@ public static List<DateTimeParseResult> DateTimeResolutionForSplit(DateTimeParse
             return results;
         }
 
+        public static DateTimeParseResult TasksModeModification(DateTimeParseResult slot, DateObject referenceTime)
```

Comment:
As static, can't it be in the task-specific class?

Reply:
Shifted code block to task-specific util file